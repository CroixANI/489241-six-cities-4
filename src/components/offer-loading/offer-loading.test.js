import React from 'react';
import renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';

import history from '../../history';
import OfferLoading from './offer-loading.jsx';

it(`Render Offer Loading component`, () => {
  const tree = renderer
    .create(
        <Router history={history}>
          <OfferLoading />
        </Router>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
