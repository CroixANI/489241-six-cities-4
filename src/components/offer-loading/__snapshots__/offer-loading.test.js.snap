// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Offer Loading component 1`] = `
Array [
  <main
    className="page__main page__main--favorites page__main--favorites-empty"
  >
    <div
      className="page__favorites-container container"
    >
      <section
        className="favorites favorites--empty"
      >
        <h1
          className="visually-hidden"
        >
          Loading...
        </h1>
        <div
          className="cities__status-wrapper tabs__content"
        >
          <b
            className="cities__status"
          >
            Please wait
          </b>
          <p
            className="cities__status-description"
          />
        </div>
      </section>
    </div>
  </main>,
  <footer
    className="footer container"
  >
    <a
      className="footer__logo-link"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="6 cities logo"
        className="footer__logo"
        height="33"
        src="/img/logo.svg"
        width="64"
      />
    </a>
  </footer>,
]
`;
